spec/constant_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe "Constant" do
+  it "Add two complex tensors." do
```

Comment:
in rspec, you can do something like:

```ruby
describe "Constant" do
  let(:graph) { Tensorflow::Graph.new }
  let(:session) { Tensorflow::Session.new }
  it "Add two complex tensors." do
    ...
    ...
```
And then you don't need repeat those instantiations in the example blocks nested within this `describe`.